### PR TITLE
CWS Support Visibility Option UI change documented

### PR DESCRIPTION
**Pull request by IanStanion-google**
Changes proposed in this pull request:

- Updated description and screenshots of the CWS dashboard UI regarding the new  changes to the Support Visibility option.

[Preview page](https://deploy-preview-4180--developer-chrome-com.netlify.app/en/docs/webstore/manage/)

### CODE REVIEW

**Comment by jpmedley** [comment]
@dasli Is this factually correct? The text needs to be right before I verify that the commas in the right place.

**Review by dasli — APPROVED** [review]
LGTM

**Comment by IanStanion-google** [comment]
@jpmedley Okay content approved by David! Feel free to do any writing-polish critique, and I will implement asap!

File: site/en/docs/webstore/manage/index.md [hunk 1]
```diff
@@ -119,9 +119,10 @@ Users will be able to communicate with you using the **Support** tab of your sto
 
 ### Using the User Support tab {: #user-support-tab }
 
-First, you must enable user feedback in the developer console Account setting page.
+Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.   
+The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
```
Comment (dotproto) [inline site/en/docs/webstore/manage/index.md:123]: ATM a single line break causes the text to render with a `<br>` inserted between the lines. We should either remove the line break to make this a single paragraph or add another line break to make "The Support Visibility option…" a distinct paragraph.

```suggestion
Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.

The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
```

**Current rendering**

Screen Shot 2022-11-07 at 12 42 32 PM

**Review by dotproto — APPROVED** [review]
Minor formatting issue to clean up, but other than that it looks good.

File: site/en/docs/webstore/manage/index.md [hunk 2]
```diff
@@ -119,9 +119,10 @@ Users will be able to communicate with you using the **Support** tab of your sto
 
 ### Using the User Support tab {: #user-support-tab }
 
-First, you must enable user feedback in the developer console Account setting page.
+Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.
```
Comment (jpmedley) [inline site/en/docs/webstore/manage/index.md:122]:
```suggestion
Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Visibility option (located in the Support tab area) must be switched on.   
```

Comment (jpmedley) [reply site/en/docs/webstore/manage/index.md:122]: Please always be strict about referring to things exactly as they're named in the interface.

File: site/en/docs/webstore/manage/index.md [hunk 3]
```diff
@@ -119,9 +119,10 @@ Users will be able to communicate with you using the **Support** tab of your sto
 
 ### Using the User Support tab {: #user-support-tab }
 
-First, you must enable user feedback in the developer console Account setting page.
+Having a Support section on your extension's Chrome Web Store page allows users to easily report issues and suggest new features separately from leaving a review. To enable user feedback on your extension, the Support Visibility option must be switched on.   
+The Support Visibility option switch is located in the Chrome Web Store Developer Dashboard, in the Support Tab section of Account Options.
 
-{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/ArMfJztL1OlP6UektUwb.png", alt="Enable User Feedback Support tab", width="800", height="185" %}
+{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/3h8A6IglCHjMN4uncopz.png", alt="Support Visibility option", width="800", height="239" %}
```
Comment (jpmedley) [inline site/en/docs/webstore/manage/index.md:125]:
```suggestion
{% Img src="image/6AZNJBRnkpQUWTKPzig99lQY8jT2/3h8A6IglCHjMN4uncopz.png", alt="Visibility option", width="800", height="239" %}
```

Comment (jpmedley) [reply site/en/docs/webstore/manage/index.md:125]: Please don't use screen captures that are unreadable. Cutting off the help doesn't hurt at all. I'd suggest something like this. Feel free to use it. 

SupportTab

**Review by jpmedley — CHANGES_REQUESTED** [review]
The paragraph over all is an improvement. The suggestions I've made are best practices for documentation.

**Comment by IanStanion-google** [comment]
@jpmedley This is cleaned up now, please approve or let me know if you notice anything else!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**